app/models/proposal.rb

```diff
@@ -143,6 +143,10 @@ def reset_status()
     end
   end
 
+  def has_subscriber?(user)
```

Comment:
yeah alias would work, or a `subscribers` method that just calls `users`...either way. I noticed in a mailer class we have in this project that there are a bunch of aliased methods and that can get confusing, but just one wouldn't be too bad.